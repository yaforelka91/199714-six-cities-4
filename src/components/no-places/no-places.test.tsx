import React from 'react';
import renderer from 'react-test-renderer';
import NoPlaces from './no-places';

const city = `Amsterdam`;

describe(`NoPlacesSnapshot`, () => {
  it(`should render NoPlaces`, () => {
    const tree = renderer.create(
        <NoPlaces city={city} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
