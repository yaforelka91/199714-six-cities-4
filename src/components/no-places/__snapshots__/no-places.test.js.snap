// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoPlacesSnapshot should render NoPlaces 1`] = `
<section
  className="cities__no-places"
>
  <div
    className="cities__status-wrapper tabs__content"
  >
    <b
      className="cities__status"
    >
      No places to stay available
    </b>
    <p
      className="cities__status-description"
    >
      We could not find any property availbale at the moment in 
    </p>
  </div>
</section>
`;
